### PR TITLE
Add model meter status.

### PR DESCRIPTION
**Pull request by cmars**

### CODE REVIEW

File: wireformat/metrics/metrics.go [hunk 1]
```diff
@@ -37,18 +37,19 @@ type Response struct {
 	NewGracePeriod time.Duration        `json:"new-grace-period"`
 }
 
+// EnvironmentResponses is a map of model UUID to wireformat responses.
```
Comment (alesstimec) [inline wireformat/metrics/metrics.go]: would this be a good time to rename Environments to Models?

Comment (cmars) [inline wireformat/metrics/metrics.go]: I was tempted...

Comment (cmars) [inline wireformat/metrics/metrics.go]: Will consider it for a followup. If we can coordinate this with a /v3/metrics endpoint, we can even rename the wireformat fields.

**Comment by cmars** [comment]
$$merge$$

**Pull request merged**